### ISSUE TITLE
Feature request: per user forums selection

### ISSUE THREAD

**Issue by ukasiu** [PayBas/RecentTopics#13]
Hi,
it'd be nice to have an option in the UCP to select forums from which posts appear in unread/recent list.

**Comment by misqos** [comment]
Hello!
Is there any chance to have it developed in the nearest future? I'd love this feature!

**Comment by PayBas** [comment]
Sorry, I can't give a timeline for this. But in the mean time, anyone is free to improve this extension and I'll look over the code.

**Comment by LtbBer** [comment]
Yes, I have to say that this is really a great idea - many times requested by our users...it would be great to have it implemented.

**Comment by ukasiu** [comment]
@LtbBer you may try https://github.com/iiet/unreadpostscontrol

**Comment by LtbBer** [comment]
Wow, it looks very promising, will try it, thanks for the tip :-)
